packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -115,22 +122,66 @@ class Accordion extends AutoControlledComponent<WithAsProp<AccordionProps>, any>
   }
 
   public static defaultProps = {
-    accessibility: defaultBehavior as Accessibility,
+    accessibility: accordionBehavior,
+    as: 'dl',
   }
 
   static autoControlledProps = ['activeIndex']
 
   static Title = AccordionTitle
   static Content = AccordionContent
 
-  getInitialAutoControlledState({ exclusive }) {
-    return { activeIndex: exclusive ? -1 : [-1] }
+  private focusHandler: ContainerFocusHandler = null
+  private itemRefs = []
+
+  actionHandlers: AccessibilityActionHandlers = {
+    moveNext: e => {
+      e.preventDefault()
+      this.focusHandler.moveNext()
+    },
+    movePrevious: e => {
+      e.preventDefault()
+      this.focusHandler.movePrevious()
+    },
+    moveFirst: e => {
+      e.preventDefault()
+      this.focusHandler.moveFirst()
+    },
+    moveLast: e => {
+      e.preventDefault()
+      this.focusHandler.moveLast()
+    },
+  }
+
+  constructor(props, context) {
+    super(props, context)
+
+    this.focusHandler = new ContainerFocusHandler(
+      () => this.props.panels.length,
+      index => {
+        this.setState({ focusedIndex: index }, () => {
```

Comment:
Can we extract this to a method? At first sight it looks like we are invoking setState in the constructor, but this is just a param to the constructor of the ContainerFocusHandler

Reply:
I just copy-pasted it from the List. Will extract it.